docs/how-to-guides/integrating-autoware/creating-vehicle-interface-package/vehicle-interface.md

```diff
@@ -0,0 +1,63 @@
+# Vehicle interface
+
+## What is the vehicle interface?
+
+The purpose of the vehicle interface package is to convert the control messages calculated from the autoware into a form that the vehicle can understand and transmit them to the vehicle (CAN, Serial message, etc.) and to decode the data coming from the vehicle and publish it through the ROS 2 topics that the autoware expects.
+So, we can say that the purposes of the vehicle_interface are:
+
+1. Converting Autoware control commands to a vehicle-specific format. Example control commands of autoware:
```

Comment:
This line ends with `Example control commands of autoware:`, but it does't have provide the example in the next line.

Reply:
done :+1: